docs/get-started/using-with-react.md

```diff
@@ -47,7 +69,23 @@ export default class App extends Component {
 
 ```
 
-Remarks
+## Using JSX with deck.gl Layers
+
+It is possible to use JSX syntax to create deck.gl layers as React children of the `DeckGL` React components, instead of providing them as ES6 class instances to the `layers` prop. There are no performance advantages to this syntax but it can allow for a more consistent, React-like coding style.
+```jsx
+  render() {
+    return (
+      <DeckGL {...viewport}>
+        <LineLayer id="line-layer" data={data} />
+      <DeckGL />
+    );
+  }
+```
+
+> Caveat: The JSX layer syntax is limitated in that it only works when the layers are direct children of the `DeckGL` component. deck.gl layers are not true React components and cannot be rendered independently by React, and the JSX support depends on deck.gl intercepting the JSX generated child elements before React tries to render them.
```

Comment:
maybe put a little `:warning:` emoji in front on "Caveat" for more attention

Reply:
didn't know emoji are supported in markdown. Does that work with both github docs and ocular?

Reply:
Definitely works in github, just need a [little tweak](https://github.com/chjj/marked/issues/233#issuecomment-323943368) for support in ocular, but deck.gl is still not ocularized right?

Reply:
True, we'd need to update the deck.gl website code. Probably not worth it right now.